archive/shanghai-2019/README.md

```diff
@@ -0,0 +1,4 @@
+# Node.js Code + Learn @ COSCon 2019
+
+- [事前准备(praparation)](./preparation.md)
+- [Dockerfile](./Dockerfile)
```

Comment:
Also add the [introduction presentation](https://docs.google.com/presentation/d/1NluODLOelMFyui10jSLY8T4M0F444LaG7Ir5u_tfOGM/edit#slide=id.p1) here?